k8s/README.md

```diff
@@ -159,32 +159,54 @@ linkerd dashboard &
 ```
 On most desktops, this will start a browser window with the dashboard. On the home screen of the dashboard under HTTP metrics you should see your namespace, zq. There is a grafana icon in the far rigght column next to zq. Click on that, and you will see some basic metrics for traffic through zqd.
 
-### Step 4: Kube State Metrics
+### Step 4: A Grafana dashboard for zqd
+Grafana is also running in the linkerd namespace: it serves the Linkerd dashboard. Now we will add our own Grafana dashboard for zqd.
 
-WIP!
+This assumes that you have run `linkerd dashboard &` and that it has port-forwarded the Grafana instance to `localhost:50750`. Use a web browser to connect to the Grafana instance running in the linkerd nampespace:
+```
+http://localhost:50750/grafana
+```
+`k8s/grafana-dashboard.json` is a sample Grafana dashboard for zqd. To import the dashboard into the local Grafana instance, click on the "+" sign on left hand side of the Grafana home screen. From the drop=down that appears, select "import". On the import screen, select upload .json file, and choose `k8s/grafana-dashboard.json`. (You may have to change the name or uid on the import screen.)
+
+To experiment with the dashboard, you may want to place some repetitive load on the zqd instance. There is a shell script `k8s/zapi-loop.sh` you can use to do a simple zq query in a loop while you watch the dashboard. This script assumes you have completed "Deploy zqd with a S3 datauri" above.
 
-## WIP: Deploying the ZQ daemon to an AWS EKS cluster
+#### Dashboard details
+The sample dashboard that includes the following PromQL queries to monitor the zqd container:
+```
+container_memory_usage_bytes{id=~"/kube.+",container="zqd"}
+rate(container_cpu_usage_seconds_total{id=~"/kube.+",container="zqd"}[1m])
+sum(rate(tcp_read_bytes_total{app_kubernetes_io_name="zqd"}[1m]))
+sum(rate(container_network_receive_bytes_total[1m]))
+```
+The linkerd metrics for the queries above are described in:
+https://linkerd.io/2/reference/proxy-metrics/
+
+The cAdvisor metrics for the queries above are described in:
+https://github.com/google/cadvisor/blob/master/docs/storage/prometheus.md
+
+## Delete the local cluster
+When you no longer need the local cluster, you can remove it with:
+```
+kind delete cluster --name zq-local
+```
 
-NOTE: this EKS procedure is not yet working end-to-end in the k8s branch!
+## Deploying the ZQ daemon to an AWS EKS cluster
```

Comment:
It's kind of a nit that I didn't get to in the previous review pass, but there's places in the readme where all caps like "ZQ" and "ZQD" is used and other places it's lowercase like "zq" and "zqd". I've generally seen us try to standardize on the lowercase representations even in titles, just because they're commands that are typed.

Reply:
Yes, I changed to lower case for all zq and zqd